Renderer.js

```diff
@@ -0,0 +1,60 @@
+
+class Renderer {
+    constructor(apiManager) {
+        this.data = apiManager
+    }
+    loadData() {
+        this.data.getUsers()
+        this.data.getQuote()
+        this.data.getPokemon()
+        this.data.getMeat()
+    }
+    renderUser() {
+        const $user = $(".user-container")
+        $user.empty()
+        const userSource = $('#user-template').html();
+        const userTemplate = Handlebars.compile(userSource);
+        const userInfo = this.data.getData().users[0]
+        const userHTML = userTemplate(userInfo);
+        $user.append(userHTML)
+    }
+    renderFriends() {
+        const $friends = $(".friends-container")
+        $friends.empty()
+        const friendsSource = $('#friends-template').html();
+        const friendsTemplate = Handlebars.compile(friendsSource);
+        const friends = [...this.data.getData().users]
+        friends.shift()
```

Comment:
This logic should be in the APIManager. here you just displaying the data, not modifying it